build.gradle

```diff
@@ -555,3 +571,25 @@ idea.project.ipr {
     }
   }
 }
+
+// Allow running diffCoverage against uncommitted files
+// See https://github.com/form-com/diff-coverage-gradle/issues/73
+ext.createDiffFile = { ->
+  def file = Files.createTempFile(URLEncoder.encode(project.name, 'UTF-8'), '.diff').toFile()
+  def diffBase = 'refs/remotes/origin/main'
+  if (System.getenv('JENKINS_HOME')) {
+    // running on Jenkins
+    diffBase = 'upstream/main'
```

Comment:
Is this upstream branch in people's forked repo?

Reply:
I think `refs/remotes/origin/main` is the main branch in the people's forked repo. `upstream/main` seems to be a special git config we have for Jenkins.

Reply:
Our internal convention is to have `origin` point to our personal fork (githubUsername/venice), and `upstream` point to the communal fork (linkedin/venice). Our dev guide mentions that briefly in passing, such as [here](https://venicedb.info/docs/dev_guide/recommended_development_workflow) but not everywhere. We should update the dev guide to say that everywhere. And IMHO, it'd be even better if the build also automatically setup the `upstream` remote.

Reply:
Luckily, we have setup this `upstream` remote as a part of Github CI. I've changed the code specifically point to upstream remote, which will be `upstream/main`.